WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressForm.swift

```diff
@@ -214,13 +214,13 @@ struct EditOrderAddressForm<ViewModel: AddressFormViewModelProtocol>: View {
                 .padding(.horizontal, insets: geometry.safeAreaInsets)
                 .background(Color(.systemBackground))
 
-                Group {
-                    TitleAndToggleRow(title: viewModel.toggleTitle, isOn: $viewModel.fields.useAsToggle)
+                if viewModel.showAlternativeUsageToggle {
+                    TitleAndToggleRow(title: viewModel.alternativeUsageToggleTitle, isOn: $viewModel.fields.useAsToggle)
                         .padding(.horizontal, Constants.horizontalPadding)
                         .padding(.vertical, Constants.verticalPadding)
+                        .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        .background(Color(.systemBackground))
```

Comment:
nit: We also have the `renderIf(condition)` modifier

Reply:
I've seen discussions about downsides of conditional modifiers. I haven't researched the topic yet, but prefer sticking to simpler approach when it fits 🙈